guides/common/modules/con_renaming-server-or-smart-proxy.adoc

```diff
@@ -1,17 +1,9 @@
-[id="Renaming_Server_or_Smart_Proxy_{context}"]
-= Renaming {ProjectServer} or {SmartProxyServerTitle}
+[id="Renaming_{project-context}_Server_or_{smart-proxy-context}_{context}"]
+= Renaming {ProjectServerTitle} or {SmartProxyServerTitle}
 
-To rename {ProjectServer} or {SmartProxyServer}, you must use the `{project-change-hostname}` script.
+To rename {ProjectServer} or {SmartProxyServer}, use the `{project-change-hostname}` script.
 
-If you rename {ProjectServer}, you must reregister all {Project} clients and configure each {SmartProxyServer} to point them to the new {Project} host name.
-If you use custom SSL certificates, you must regenerate them with the new host name.
-If you use virt-who, you must update the virt-who configuration files with the new host name.
-
-If you rename {SmartProxyServer}, you must reregister all {SmartProxy} clients and update the {SmartProxy} host name in the {ProjectWebUI}.
-If you use custom SSL certificates, you must regenerate them with the new host name.
-
-[WARNING]
+[IMPORTANT]
 ====
-The renaming process shuts down all {ProjectServer} services on the host being renamed.
-When the renaming is complete, all services are restarted.
+When changing the DNS domain name of your {ProjectServer} or {SmartProxyServer}, consider updating the hostname using `{project-change-hostname}` to avoid networking issues.
```

Comment:
I am not a native English speaker, and "consider" is not listed in [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119), but I read this sentence as:
When changing the domain name, it is a good idea but not an absolute must (= "SHOULD" in RFC2119 terms) to use project-change-hostname.

However, I think it is an absolute MUST to perform this when changing the domain name (@sayan3296 might comment as the original author of the BZ)?

BTW, I think "DNS domain name" is rather doubled and "domain name" is perfectly sufficient in this case.

Reply:
Thanks for having a look @evgeni.

"Consider doing" does indeed correspond to SHOULD as defined in RFC2119. It's also how the reporter of the BZ phrased the requirement.

MUST would indeed be a totally different story, so I'm happy to rephrase the sentence if needed.

Reply:
@evgeni I didn't manage to get hold of Sayan, but based on your recommendation, I rephrased the sentence as a requirement (MUST)  after all in the latest commit: https://github.com/theforeman/foreman-documentation/pull/2231/commits/b636fe748677f2baf15639a59010a573493c1592

Does this look okay to you?

Reply:
@asteflova 
I was on leave yesterday and so I wasn't able to reply back. 
I don't want to pick between `should` and `must` but if I understand the outcome of the `hostname` or `domain` change correctly, Then the satellite\capsule\foreman would not remain properly functional if `satellite-change-hostname` is not being used. 
And regarding the commit you pointed out, I read the statement and it has the necessary clarity.